### PR TITLE
Weather card without chart

### PR DESCRIPTION
**Pull request by c727**
backend PR: https://github.com/home-assistant/home-assistant/pull/14076

fix: #999 
fix: #464 

current:
current

PR:
unbenannt

### CODE REVIEW

**Comment by c727** [comment]
We could also use these icons (with or without animation) but they are not 100% material https://www.amcharts.com/free-animated-svg-weather-icons/
icons

**Comment by arsaboo** [comment]
Super excited about this 👍 

Wondering if user can configure whether to use the animated icons or the static ones for the weather card.

File: src/cards/ha-weather-card.html [hunk 1]
```diff
@@ -1,97 +1,167 @@
...
+              <iron-icon icon="mdi:water-percent"></iron-icon>
+              <div>[[localize('attribute.weather.humidity')]]</div>
             </div>
-            <div class="condsmall" hidden$="[[!attr.humidity]]">
-              [[localize('attribute.weather.humidity')]]: [[attr.humidity]]%
+            <div>[[stateObj.attributes.humidity]] %</div>
+          </div>
+          <div class="attribute flex">
+            <div class="icon-container">
+              <iron-icon icon="mdi:flag-variant"></iron-icon>
```
Comment (arsaboo) [inline src/cards/ha-weather-card.html]: `mdi:weather-windy` may be a better icon for wind_speed.

File: src/translations/en.json [hunk 2]
```diff
@@ -1,8 +1,10 @@
 {
   "attribute": {
     "weather": {
+      "air_pressure": "Air pressure",
       "humidity": "Humidity",
       "visibility": "Visibility",
+      "wind_bearing": "[\"N\", \"NNE\", \"NE\", \"ENE\", \"E\", \"ESE\", \"SE\", \"SSE\", \"S\", \"SSW\", \"SW\", \"WSW\", \"W\", \"WNW\", \"NW\", \"NNW\", \"N\"]",
```
Comment (c727) [inline src/translations/en.json]: is this OK?

**Comment by balloob** [comment]
The middlepart looks messy and cramped. The measurements are not aligned with icon/label. We need a better way.

**Comment by c727** [comment]
what about this? suggestions are welcome
this

**Comment by c727** [comment]
without text

**Comment by arsaboo** [comment]
I like no. 2.

**Comment by pepeEL** [comment]
For me good is no. 1 and no. 2

**Comment by balloob** [comment]
The challenge with 3 is the lack of description. 2 is still chaotic. Every row has it's own column section. I prefer 1.

make sure btw that everything on the left is aligned. Look how the table starts further in than the icon and the icon starts further in than the header. It should all be starting at the same line.

**Comment by balloob** [comment]
Thinking about it more, maybe the humidity, pressure etc are not important enough to be in the card. We can move them to the more info ?

**Comment by raccettura** [comment]
I'd say humidity is pretty important for a decent sized part of the world where heat index is a normal part of the forecast during warmer months.  Pressure IMHO is not terribly important for most people.

**Comment by c727** [comment]
new design in first post, forecast has max. 7 days

**Comment by arsaboo** [comment]
Looks good. Can we fix the alignment of the text on the right to be closer to your post [here](https://github.com/home-assistant/home-assistant-polymer/pull/1117#issuecomment-383446641)

**Comment by arsaboo** [comment]
Also, for English, should we use the standard 3 letters for days (Mon, Tue, etc.) now that we have some space?

**Comment by c727** [comment]
...this is German

**Comment by c727** [comment]
please check if you agree with my changes in translation file

more-info

**Comment by fanthos** [comment]
I like the new design mostly, but the table style does not work well with platform that has per-hour forecast providers, like OpenWeatherMap.
#375 #373

**Comment by sgttrs** [comment]
Maybe visibility can also be hidden?

**Comment by c727** [comment]
Can we focus the discussions on major issues as fanthos posted, please

**Comment by fanthos** [comment]
I think the per-hour works better in more cases. The daily forecast convert could be done in frontend if needed.

**Comment by c727** [comment]
IMO if you have both, temp and tempLow, a graph makes no sense
yahoo and BR have this for example

**Comment by c727** [comment]
finished :)

final

**Review by balloob — APPROVED** [review]

**Pull request merged**